Apps/Sandcastle/gallery/development/3D Models Instancing 3D Tiles.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
Can't we just use the cities.html example for this?  Just point it to a tiles.json with i3dm tiles?